docs/concepts.md

```diff
@@ -32,7 +32,7 @@ or reference implementations.
 
 ## Inferno Template
 
-A template for writing Inferno Test Kits.
+A template for writing Inferno Test Kits. See [Template Layout](http://localhost:4000/docs/getting-started/repo-layout-and-organization) for more information.
```

Comment:
Shouldn't be 'localhost'.  Just make it a relative link.

Reply:
I think there are some other `localhost` links throughout the site. I assumed that was on purpose. I can go through and make them all relative.

Reply:
Nope, just a mistake!